### ISSUE TITLE
Hardware/training time/epochs etc

### ISSUE THREAD

**Issue by wamiq-reyaz** [nv-tlabs/ATISS#9]
Hello,

I was wondering what is the expected time to train - the paper only mentions that you choose the best model from a very large number of iterations. What hardware did you use to train the model on and how long does that usually take, just to get a ballpark figure?

In the same vein, did you (or coauthors) use a different lr schedule, optimizer etc that seems to work better?

PS: Do you have any intentions of releasing the Kaolin scripts used to render the figures in the paper. They look very very nice!

**Comment by wamiq-reyaz** [comment]
A follow-up would be what are the loss ranges for the size, translation, angle, and label that you expect or report the figures in the paper for?

**Comment by paschalidoud** [comment]
Hi @wamiq-reyaz,

We run all our experiments on an NVIDIA GeForce GTX 1080 Ti GPU. Training lasted on average around 1-1.5 days. We experimented with different lr scheduling and optimizers such as RAdam, but we didn't experience any difference in performance, thus we ended up using a learning rate of 1e-4 without weight decay.

Regarding loss ranges, for the trained model that we used for generating the bedroom scenes in the paper, the ranges that we observed for each loss term are summarized below:

- label loss: 1.235 - 0.9793
- size NLL loss: 12.48 - 3.042
- rotation NLL loss: 4.773 - 0.4792
- translation NLL loss: 14.301 - 5.832

Please note that the above numbers can vary among runs.

Best,
Despoina

**Comment by paschalidoud** [comment]
@wamiq-reyaz, I am closing this issue for now, but please feel free to reopen in case you have more questions!

Best,
Despoina

**Comment by paschalidoud** [comment]
@wamiq-reyaz, I am closing this issue for now, but please feel free to reopen in case you have more questions!

Best,
Despoina

**Comment by paschalidoud** [comment]
@wamiq-reyaz, I am closing this issue for now, but please feel free to reopen in case you have more questions!

Best,
Despoina

**Comment by Jingyu6** [comment]
> Hi @wamiq-reyaz,
> 
> We run all our experiments on an NVIDIA GeForce GTX 1080 Ti GPU. Training lasted on average around 1-1.5 days. We experimented with different lr scheduling and optimizers such as RAdam, but we didn't experience any difference in performance, thus we ended up using a learning rate of 1e-4 without weight decay.
> 
> Regarding loss ranges, for the trained model that we used for generating the bedroom scenes in the paper, the ranges that we observed for each loss term are summarized below:
> 
> * label loss: 1.235 - 0.9793
> * size NLL loss: 12.48 - 3.042
> * rotation NLL loss: 4.773 - 0.4792
> * translation NLL loss: 14.301 - 5.832
> 
> Please note that the above numbers can vary among runs.
> 
> Best, Despoina

Hi @paschalidoud, would you mind also sharing the validation stats for this? When I train the model (despite on the new version of the dataset), it seems to overfit the training data quite easily so I want to see whats stats did you get when you train the ones used in the paper? Thanks a lot in advance!

Best,
Jingyu
